nursery/parse-pe-export-data.yml

```diff
@@ -0,0 +1,33 @@
+rule:
+  meta:
+    name: find RVA from a function by parsing the PE Export Data
+    namespace: load-code/pe
+    author: sara-rn
+    scope: function
+    examples:
+      - 73CE04892E5F39EC82B00C02FC04C70F
+  features:
+    - and:
+      - os: windows
+      - or:
+        - characteristic: loop
+        - mnemonic: movzx
```

Comment:
could this also be `mov`?

Reply:
I have seen movzx in 2 samples so far.
mov     eax, [ebp+addressofNamesOrd]
movzx   eax, word ptr [eax+ebx*2]
In the chapter 19 of PMA the same is done with mov: mov  cx, word ptr [ebx+ecx*2]

Reply:
by removing the movz there will be FP I believe but maybe there is less chance to miss hits, let me know what you think

Reply:
sorry, I wasn't clear, I meant to suggest to add `mov`?